Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
 resolver = "2"
 
-members = ["issuance_contract", "did_contract", "deployer_contract", "vault_contract"]
+members = ["issuance_contract", "did_contract", "deployer_contract", "vault_contract", "verifiable_credentials_contract"]
```

Comment:
@MarioRodriguezS IMO, a suitable contract name would be `vc_issuance_contract`. WDYT?